app/.eslintrc.js

```diff
@@ -3,14 +3,8 @@ module.exports = {
   extends: '@react-native-community',
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint'],
-  overrides: [
-    {
-      files: ['*.ts', '*.tsx'],
-      rules: {
-        '@typescript-eslint/no-shadow': ['error'],
```

Comment:
Why remove this? Is this the same as removing the `no-shadow` below?

Reply:
I wasn't sure why these defaults were here to be honest. Removing the whole `overrides` section makes our linting stricter (better, imo).

Reply:
Ok. Do you know if `no-shadow` rule is enabled by default? I think we need to turn it on if it's disabled after removing this override.

Reply:
It was, but I just pushed an addition that makes it error instead of warn on shadowing.

Reply:
Oops, I think I get (some) of these defaults now. I think that `@typescript-eslint/no-shadow` has different rules than `no-shadow` (which since it's not prefixed I'm assuming is from the base `eslint` rules). So what the RN team did is basically prefer `@typescript-eslint/no-shadow` over `eslint/no-shadow` for `.ts` and `.tsx` files specifically. I think I'm going to revert the `*no-shadow` changes but I'd still like to keep the `no-undef` changes. Sorry for the back-and-forth.

Reply:
Ok no problem